src/pt-functions.lisp

```diff
@@ -170,6 +170,24 @@ in which case create a new COLOR object as the return value."
 
 
 
+;;; Font Loading and Text Drawing Functions (Module: text)
+
+;; Text font info functions
+
+(defun measure-text-ex (text)
+  "Returns an RL-VECTOR2 with the width (x) and height (y) of the TEXT object. This accounts for
+font, size and spacing."
+  (check-type text text)
+  (let ((retval (make-vector2 0 0)))
+    (claylib/ll:measure-text-ex (c-struct retval)
+                                (c-struct (font text))
+                                (text text)
+                                (size text)
+                                (spacing text))
+    retval))
```

Comment:
> we should allow the user to pass their own

Yeah that makes sense.

> Correct me if I'm wrong but you should be able to define this with `defun-pt`.

I thought not because `MeasureTextEx` takes a different number of args than `measure-text-ex`. i.e. to make `measure-text-ex` more claylib-y I made it take a `text` object. Maybe I understand `defun-pt` incorrectly though, I thought the args had to match the raylib args.

> Also, you think measure-text-ex makes sense to the user? I was going to say just call it `measure-text`

Maybe renaming to `measure-text-object` would be better?

Initially I wanted to replace `measure-text` with what is now `measure-text-ex` but as you say, they do things slightly differently. As of now `measure-text` is just `claylib/ll:measure-text`, which returns only the width of a string with the default font and spacing. `measure-text-ex` now conveniently takes a claylib `text` object and gives its height & width according to its font, spacing & size.

Maybe I should just replace `measure-text` with `measure-text-object` and when one needs to measure stuff before initializing an object one can use `claylib/ll:measure-text` or `claylib/ll:measure-text-ex`. How does that sound?